src/mame/drivers/hp9k_3xx.cpp

```diff
@@ -469,6 +546,8 @@ MACHINE_CONFIG_START(hp9k3xx_state::hp9k300)
 	MCFG_MCS48_PORT_P2_OUT_CB(WRITE8(*this, hp9k3xx_state, iocpu_port2_w))
 	MCFG_MCS48_PORT_P1_IN_CB(READ8(*this, hp9k3xx_state, iocpu_port1_r))
 	MCFG_MCS48_PORT_T0_IN_CB(READ8(*this, hp9k3xx_state, iocpu_test0_r))
+	MCFG_MCS48_PORT_P2_IN_CB(READ8(*this, hp9k3xx_state, iocpu_port2_r))
+	MCFG_MCS48_PORT_T1_IN_CB(READ8(*this, hp9k3xx_state, iocpu_test1_r))
```

Comment:
Using `CONSTANT` here would be simpler than the `READ8` trampolines that return constants.

Reply:
T0 and T1 shouldn't really be `READ8` anyway, since they're individual lines.